docker/entrypoint.bash

```diff
@@ -67,6 +66,13 @@ else
     chown -R "$NB_USER" "${DY_SIDECAR_PATH_OUTPUTS}"
 fi
 
+mv "${NOTEBOOK_BASE_DIR}/README.ipynb" "${NOTEBOOK_BASE_DIR}/workspace/README.ipynb"
+
+echo "Creating /home/${NB_USER}/.virtual_documents"
+mkdir -p "/home/${NB_USER}/.virtual_documents"
+chown -R "$NB_USER" "/home/${NB_USER}/.virtual_documents"
+export JP_LSP_VIRTUAL_DIR=/home/${NB_USER}/.virtual_documents
```

Comment:
what is this export for? should that not be set in the Dockerfile?

Reply:
Managed to move it to the Dockerfile. Read more on usage [here](https://jupyterlab-lsp.readthedocs.io/en/latest/Configuring.html#virtual-documents-dir). Had to move it to avoid permissions issues.